setup.py

```diff
@@ -17,9 +17,9 @@ def read(fname):
     "napari>=0.4.9",
     "napari-plugin-engine>=0.1.4",
     "numpy",
-    "aicssegmentation >= 0.3.0",
+    "aicssegmentation == 0.4.0",
     "magicgui >= 0.2.9",
-    "aicsimageio>=3.3.6,<4",
+    "aicsimageio == 4.1.0",
```

Comment:
I think it would be better to use `>=` instead of `==` since this makes it seem like aicssegmentation 0.4.1 might break it, for example

Reply:
Even better would be `~=` unless others think strict equality is best for some reason